main.go

```diff
@@ -53,8 +53,8 @@ func initConfig() {
 	}
 	viper.SetConfigType("yaml")
 
-	if err := viper.ReadInConfig(); err == nil {
-		log.Warn().Msgf("Using config file: %v", viper.ConfigFileUsed())
+	if err := viper.ReadInConfig(); err != nil {
```

Comment:
With these changes, when the indexer is launched without explicitly specifying the config, the logs do not show which config is being used

Before:
```
[~/dev/aurora/aurora-relayer-indexer]$ go run . -k true
{"level":"warn","time":"2023-01-26T16:30:47+01:00","message":"Using config file: /aurora-relayer-indexer/config/local.yaml"}
```


After:

```
$ go run . -k true
{"level":"info","time":"2023-01-26T16:30:11+01:00","message":"67847148"}
{"level":"info","time":"2023-01-26T16:30:11+01:00","message":"67847149"}
{"level":"info","time":"2023-01-26T16:30:11+01:00","message":"67847150"}
{"level":"info","time":"2023-01-26T16:30:11+01:00","message":"67847151"}
{"level":"info","time":"2023-01-26T16:30:11+01:00","message":"67847152"}
```